packages/plugin-flex/package.json

```diff
@@ -39,15 +39,15 @@
     "prepack": "./bin/prepack",
     "postpack": "rm -f oclif.manifest.json",
     "replace-doc": "cp docs/README.md README.md && oclif-dev readme",
-    "test": "cd ../.. && jest packages/flex-plugin-webpack --color",
-    "test:watch": "cd ../.. && jest packages/flex-plugin-webpack --watch --color"
```

Comment:
That's terrifying - so we haven't been testing plugin-flex?

Reply:
Luckily the root test command just does a star on all packages and skips this script!  I don't think we are using this individually for anything right now.

Reply:
So we HAVE been testing plugin-flex

Reply:
okay cool. You need to also change how profile/accountSid/secret etc is read - that is the biggest breaking change for this version.